src/pcp.rs

```diff
@@ -329,69 +330,73 @@ pub trait Gadget<F: FieldElement> {
 
     /// Returns the circuit's arithmetic degree. This determines the minimum length the `outp`
     /// buffer passed to `call_poly`.
-    fn deg(&self) -> usize;
+    fn degree(&self) -> usize;
 
     /// This call is used to downcast a `Box<dyn Gadget<F>>` to a concrete type.
     fn as_any(&mut self) -> &mut dyn Any;
 }
 
 /// Generate a proof of an input's validity.
-pub fn prove<F, V>(x: &V, joint_rand: &[F]) -> Result<Proof<F>, PcpError>
+pub fn prove<F, V>(input: &V, joint_rand: &[F]) -> Result<Proof<F>, PcpError>
 where
     F: FieldElement,
     V: Value<F>,
 {
-    let g_calls = x.valid_gadget_calls();
+    let gadget_calls = input.valid_gadget_calls();
 
-    let mut shim = x
+    let mut shim = input
         .gadget()
         .into_iter()
         .enumerate()
-        .map(|(idx, g)| Ok(Box::new(ProveShimGadget::new(g, g_calls[idx])?) as Box<dyn Gadget<F>>))
+        .map(|(idx, g)| {
+            Ok(Box::new(ProveShimGadget::new(g, gadget_calls[idx])?) as Box<dyn Gadget<F>>)
+        })
         .collect::<Result<Vec<_>, PcpError>>()?;
 
     // Create a buffer for storing the proof. The buffer is longer than the proof itself; the extra
     // length is to accommodate the computation of each gadget polynomial.
     let data_len = (0..shim.len())
-        .map(|idx| shim[idx].arity() + shim[idx].deg() * (1 + g_calls[idx]).next_power_of_two())
+        .map(|idx| {
+            shim[idx].arity() + shim[idx].degree() * (1 + gadget_calls[idx]).next_power_of_two()
+        })
         .sum();
     let mut data = vec![F::zero(); data_len];
 
     // Run the validity circuit with a sequence of "shim" gadgets that record the value of each
     // input wire of each gadget evaluation. These values are used to construct the wire
     // polynomials for each gadget in the next step.
-    let _ = x.valid(&mut shim, joint_rand);
+    let _ = input.valid(&mut shim, joint_rand);
 
     // Fill the buffer with the proof. `proof_len` keeps track of the amount of data written to the
     // buffer so far.
     let mut proof_len = 0;
     for idx in 0..shim.len() {
-        let g = shim[idx]
+        let proof_gadget = shim[idx]
```

Comment:
`gadget` is fine IMO. If you want to give this a more specific name than `gadget`, I would go with `prove_shim_gadget`, since it's snake case version of `ProveShimGadget`. `proof_gadget` is a new word, and it's a little confusing. You would also want to make a similar change in `verify()`.

Reply:
There's no `fn verify` but I double checked `fn prove` and `fn query` for consistency.